Replace Vehicle Excise Duty and Fuel Duty 2

Xyleneb opened the pull request:
I'm trying it again!

Main changes from first edition:

1. Removed the ban on the sale of petrol cars by 2029, which involved getting the public on board with it in 12 years and also involved looking past the fact that you could import a petrol car from France or Germany, at which point the rule only deprives *UK* dealerships of them.

2. Changed the tax criteria from vehicle weight to power output. Yes, it's still based on anticipated use rather than actual use. Please see 'Transport Debate Needed' on why this is the case.

Why was power output chosen specifically? Well in summary because it's easier, but it'd be easier still to explain why I didn't choose the rest:

Tax on Weight:
Vehicle weight is constant per vehicle, but varies a lot between vehicles. The line in the sand would seem fairly arbitrary. Plus heavier cars are on average 25% safer. It's fine if weight shedding is used as part of a broader package of reaching better vehicle efficiency, but to encourage it through the tax system is perhaps unwise.

Tax on Power Capacity:
This is the equivalent of having vehicles with arbitrarily small tanks in them, just to avoid the taxes. You could easily fit a bigger tank, or in the case of electric vehicles a bigger battery reserve. It seemed pointless given this point to put a tax on vehicle power reserves based on how they rolled out of the factory.

Tax on Tyre Wear:
The best way to accurately measure how much your car has used the roads is to look at the tyres. Each time you buy fresh tyres, you would have to pay a lot of tax. When you hand in your old ones and they're shown to be not very worn, then you'd get a rebate. So in a way it encourages you to get fresh rubbers frequently. Safe, right? Well, even if you *forced* the poor to get fresh rubbers at the expense of food or rent, what is to stop those slightly better off from buying untaxed rubber overseas? You would need at the very least EU-wide consensus on car taxation. This option was ruled out.

Tax on Vehicle Class:
Some cars are recreational more than functional. Some cars are naturally heavier and cause more surface wear. But what about saloon cars or utility vehicles? Are they for work or pleasure? Will our tax system stay competitive with the rest of the world if someone could simply buy a sports car from France? It's yet another arbitrary line in the sand which no better relfects road wear than anything else.

Tax on Anticipated Road Wear:
The problem with a tax on this is that it's really hard to anticipate. Yes, HGVs do more damage, but does your business use them four hours a week or 24/7? Without a black box I couldn't know - I would have to take your word for it. Let me guess, you barely used it at all...

So here's why I chose Power Output:
Our current system is already based around power output. It's just a simple case of changing bhp to kilowatts. Powerful vehicles *tend* to be bigger, heavier, and cause more road wear than their little-powered cousins. The disadvantage to a power output tax was that, in Japan it was considered to stifle competitiveness in their design department. This was under the very unique situation that the Japanese tax system had created though. If ours is fairer, it need not punish too severely.

What says everyone?
Can I get this passed or what?

Autumn-Leah commented:
What was your reasoning for removing the "No new fossil-fuel powered vehicles may be sold in the UK after 2029" line?

Autumn-Leah commented:
oh, my bad, I'll delete these

Autumn-Leah commented:
Yeah sure, I see no problem with it. Aside from taxation being theft and government being lame at taking tax funds and distributing them in a somewhat intelligent manner, but this party operates on the premise of benevolent statism and taxation being theft is a whole other topic. Nothing specifically wrong with the policy unless I attempt to remove *all taxation* from the manifesto. Have an upvote.

Vote: ✅

philipjohn commented:
Removing this bit alone I'm afraid makes it a no for me:

> No new fossil-fuel powered vehicles may be sold in the UK after 2029.

That's a good policy, already considered or being passed by countries *today*. We **have** to move away from fossil fuels and banning fossil fuel-powered cars is a big step forward in that direction.

The rest of the proposal, for me, suffers from the same basic premise: that the roads only benefit those who use them.

Commerce relies on good transport, including roads. Thousands of people rely on companies being able to send lorries and vans up and down the country to deliver goods. Those companies are going to have to charge more either for the products or for delivery to meet the additional cost associated with this tax change.

People who cannot afford their own transport or are unable to have their own transport will be the ones who end up paying for this, because business will pass on the cost to them. This could well disproportionately affect those on low incomes and people with low/no mobility.

Consider imports and exports too. Post-Brexit it'll be more expensive to move goods in and out of the country as it is, without then taxing the fuel used by the lorries transporting those good. Yet another reason for businesses to move their operations to the European mainland.

Good infrastructure - whether that be roads, rail, telecommunications - is vital to a well functioning economy. Well kept roads benefit everyone, so taxing only those who use them, based on usage, undervalues their importance.

Infrastructure should be funded through general taxation because everyone benefits, so everyone who can should contribute.

Vote: ❎

Autumn-Leah commented:
Actually, Phil has a point

Vote: ❎

Xyleneb commented:
>Yeah sure, I see no problem with it. Aside from taxation being theft and government being lame at taking tax funds and distributing them in a somewhat intelligent manner, but this party operates on the premise of benevolent statism and taxation being theft is a whole other topic. Nothing specifically wrong with the policy unless I attempt to remove all taxation from the manifesto. Have an upvote.

I thought you barked up the Marx tree more than the Bakunin tree... Well, I can't abolish taxation overnight nor establish a mutualist society overnight. Doing that sort of thing will take some time.

>That's a good policy, already considered or being passed by countries today. We have to move away from fossil fuels and banning fossil fuel-powered cars is a big step forward in that direction.

I think it's in the realm of impossible, undesirable and unelectable. But I could add it back in and leave it on the books to get the proposal passed. Only, it doesn't seem to be your only gripe with it.

>Commerce relies on good transport, including roads. Thousands of people rely on companies being able to send lorries and vans up and down the country to deliver goods. Those companies are going to have to charge more either for the products or for delivery to meet the additional cost associated with this tax change.

So, in a way, those who benefit from the roads _indirectly_ will still be _paying for them_ indirectly?

>People who cannot afford their own transport or are unable to have their own transport will be the ones who end up paying for this, because businesses will pass on the cost to them. This could well disproportionately affect those on low incomes and people with low/no mobility.

Subsidies apply for businesses, and economies of scale also apply to these businesses. I do not fear the impact upon the poor, if anything fuel duty is more of a problem in terms of limiting their mobility.

>Consider imports and exports too. Post-Brexit it'll be more expensive to move goods in and out of the country as it is, without then taxing the fuel used by the lorries transporting those goods. Yet another reason for businesses to move their operations to the European mainland.

Vehicle registration is fairly expensive all across the EU. In terms of taxing the fuel, your current policy is to regulate that fuel into oblivion.

>Good infrastructure - whether that be roads, rail, telecommunications - is vital to a well functioning economy. Well kept roads benefit everyone, so taxing only those who use them, based on usage, undervalues their importance.

Make it clear to me: Do you want people who can't afford a car to pay for roads? Earlier it was yes, then it was no, and now it's yes again.

>Infrastructure should be funded through general taxation because everyone benefits, so everyone who can should contribute.

If you scrap VED, either no one will contribute, or they *all* will when you hike up everyone's general tax bill to cover the £10 billion hole. How the poor will thank us! More frustratingly than that, I don't see what I could possibly do to this submission to have it meet your approval. Scrutiny is fine, it makes the best policy... But unlike all the other ideas that other people have had before me and I've just submitted them, this one I had to craft myself, out of reading reports and original thought. So I'd really love to know what I could do with it just to get it to a point where people can stomach it. It's annoying because what's on the books sucks already, and there are far more contentious things than this that I could submit instead of this.


 but this one I had to champion or craft with a chisel.

Autumn-Leah commented:
@Xyleneb Just because I'm an ancom it doesn't mean I can't say things typically said by ancaps. Taxation *is* theft under the current statist capitalist system, so whilst we still have capitalism and statism, I'd rather people not be coerced to give their resources to the state, as that will only slow it's downfall.

Autumn-Leah commented:
If you add back in "No new fossil-fuel powered vehicles may be sold in the UK after 2029." I'll vote to approve it btw

Xyleneb commented:
>Just because I'm an ancom it doesn't mean I can't say things typically said by ancaps. Taxation is theft under the current statist capitalist system, so whilst we still have capitalism and statism, I'd rather people not be coerced to give their resources to the state, as that will only slow it's downfall.

If one is red and the other yellow, I'm orange. Also a reformist moreso than revolutionary. No grande downfalls for me!

>If you add back in ... I'll vote to approve it btw.

Well, if I distill phil's refutations down to the basics they appear to be the following:
1. Too punishing for HGVs and logistics companies.
2. The tax money should come from general taxation, not specifically targeted taxation.

The former point I don't really understand because these companies could and should be paying their fair share for the roads they use and the damage they create. I don't see it pushing the cost of a bit of essential kit off Amazon up by very much either. The overheads for logistics companies are high from paying high fuel duty and for paying for unnecessary training for HGV drivers. It used to be the case that your driver's license covered anything up to 7.5 tonnes. Now I don't think that's the case. In addition you need to get 'CPC' training and a digital tachograph card, so if you think you might want to become a HGV driver then it'll cost you a grand just to find out.

The latter point I don't really understand either. No commitment to ring-fence tax is contained herein. So, what's left is an argument over whether some kind of 'car tax' should exist at all, or if the tax man should collect a "general tax" from somewhere else. I don't want the tax man to be able to turn up at your door and demand more money "because we need more". How it is "spent" can be general, but how it is collected mustn't be.

Autumn-Leah commented:
I fail to see how Phil's objections mean you can't cater to my objection. I'm fine with the rest of the policy, I just want that paragraph added in.

Autumn-Leah commented:
I fail to see how Phil's objections mean you can't cater to my objection. I'm fine with the rest of the policy, I just want that paragraph added in.

Xyleneb commented:
>I fail to see how Phil's objections mean you can't cater to my objection. I'm fine with the rest of the policy, I just want that paragraph added in.

Well in a way it's a team sport, scrutiny makes the best policy and simple dismissals comparatively don't hold much value. I guess I should apologise for thinking aloud...

The main changes are now added.

Note that I've changed the wording a bit. "No *new* cars beyond 2029" would lead to people saying "I guess we can just import *last year's* models then". So that loophole is closed. Instead, I've opened another one up by allowing "alternative fuels" that are still fossil-based. You can sell a combustion engine if it burns hydrogen, kerosene, propane or bio-fuels.

Will that do?

I think the wording is still a tad off but you can always work on that later. Spelling and grammar mistakes will get pulled out of there eventually.

philipjohn commented:
> I think it's in the realm of impossible, undesirable and unelectable.

That governments *are* doing it shows it clearly is possible, desirable, and electable.

> So, in a way, those who benefit from the roads indirectly will still be paying for them indirectly?

Yes. The whole country benefits from the economic benefits of a well-maintained transport system.

> Subsidies apply for businesses, and economies of scale also apply to these businesses.

The local community transport scheme doesn't benefit from many economies of scale when it helps those with limited mobility to be included in the community. Nor does the local bus service that represents a lifeline to many. Volunteers who collect donations for the local food bank don't have any subsidies to apply for.

> In terms of taxing the fuel, your current policy is to regulate that fuel into oblivion.

Deliberately and proudly so!

> Make it clear to me: Do you want people who can't afford a car to pay for roads? Earlier it was yes, then it was no, and now it's yes again.

I've been consistent that general taxation should pay for the roads, and given that the Manifesto also states that no-one should be taxed until they earn a living wage, no-one would pay those taxes unless they could afford to.

> More frustratingly than that, I don't see what I could possibly do to this submission to have it meet your approval. Scrutiny is fine, it makes the best policy... 

It's​unlikely we're going to agree on this one. The basic premise for me is flawed.

Vote: ❎

Autumn-Leah commented:
It's still a no, as your alteration isn't even close to what I suggested that the alteration should be.

Vote: ❎

Xyleneb commented:
>It's still a no, as your alteration isn't even close to what I suggested that the alteration should be.

I was sure I included them at the time. What am I missing?

Autumn-Leah commented:
"No new fossil-fuel powered vehicles may be sold by a business in or imported for the sole purpose of regulated or unregulated or taxed or untaxed resale in the UK after 2029. Reselling cars that were present in the UK prior to 2029 by an individual will remain legal and will be discouraged with scrapping incentives"

Was my suggestion, you put:

"No petrol or petro-diesel powered vehicle may be sold or imported for the purposes of resale by businesses that bare a date of manufacture later than the year 2029."

Xyleneb commented:
By including the word "new" you left a loophole open to sell petrol cars in the year '2035' for example so long as they are from 2034. I'm saying; no, you can't import last year's cars and call them "old" and therefore legal. We'd end up Europe's dumping ground for last year's car models. It'd be completely counter-productive.

I could have removed the word "new" and left it at that. This would have caused a few things though:

By rewording it, I'm saying you don't have to offload your entire fleet of vehicles by 2028. You can do it gradually.

Also, I'm saying that it will still be legal for rare car auctions to carry on their businesses so long as their vehicles do not date past 2028.

Antiques and heritage get to remain in the country. HGV companies aren't faced with the prospect of replacing the entire fleet in one massive cost. The downside is that used car dealers could continue to import petrol cars. But those cars will remain stuck in 2028 and earlier. At some point the consumer will be faced with the choice of clapped out 10 year old models with emissions taxes, or modern electrified luxury.

I kept the part about scrapping schemes because it's already concise, clear and in keeping with the aims.

This is the best that I could hope to do unless the wording I put in there is still missing something that I'm not seeing. I think as well that the status quo has many unaddressed problems with it, which I have tried to consider (such as consumers being unable to do anything with their Triumph Dolomite other than scrap it come 2029).

Autumn-Leah commented:
"No fossil-fuel powered vehicles, with a manufacture date later than 2029, may be sold by a business in (or imported for the sole purpose of regulated or unregulated or taxed or untaxed resale in) the UK after 2029. Reselling cars that were present in the UK prior to 2029 by a individuals and businesses will remain legal and will be discouraged with scrapping incentives."

This (adjusted version) means:

* Businesses can't sell fossil-fuel powered vehicles made after 2029
* Individuals can't sell fossil-fuel powered vehicles made after 2029 if it was brought in for the sole purpose of sale, but can if it was their car in another country and took it here to use as their car (it'd be fairly obvious if this was the case or not)
* Businesses and individuals can resell old cars (allowing retro auctions and used car resale)

HGV companies should have the sense to replace the fleet gradually if they can't do it in one go.

> At some point the consumer will be faced with the choice of clapped out 10 year old models with emissions taxes, or modern electrified luxury.

I highly doubt this will happen, given the advancement of technology, and the push for electric cars in the car industry, other governments, and in NGOs, that's 12 years for electric cars to become cheaper, and 22 until your situation would apparently play out, worst case scenario, the gov has to subsidise peoples cars, or make public transport better, or encourage people to use ridesharing services, start carpooling, or pretty much any other solution. Additionally, the adjustment would allow for a slow stream of newer used cars to enter the market. Also, 10 year old cars aren't that bad (they definitely won't be in 22 years) and imo carbon emission charges should be adjusted for income and not apply to those who are financially unable to upgrade. If Something New has a emmission charge that isn't adjusted for income on the manifesto then I need to go sort that.

LudovicD commented:
Modern vehicles are a lot more expensive than old fossile fuel ones, also there is not a 400-mile battery (yet) for electrified vehicle, so we will need hybrid for a while. We really to need to really bring down the cost of zero-emission propulsion, and tax carbon emissions according to the net warming it creates (net warming = warning minus the reduced consumption of heating fuel due to the increase temperature of the warming created by the non-heating fuel) (the warming of global warming does reduce our consumption of heating fuel, at least in country as cold as ours). I support smart taxes & electric charging points in every district postcode/electoral ward, but a firm law to forbid ALL fossile fuel car, it has to progress with out-the-box thinking, the tranistion has to be started in a strong kick-start way and then spread out over a full 12 and a 1/2 years (150 months). But first we need a decent offer of suitable vehicles, at a reasonable price point, i mean the price point a median person would be able to buy which is about the price of a 7-year old 2nd hand average compact car (the offer is extremely reduced at point) (there are some 7-year old Prius, but that's about it). And we also need to ban the non-hybrid vehicles a long long time (a generation & a bit) before banning the (non-pluggable) hybrid ones, and them a long long time before banning the pluggable hybrids. in the short term, i am in favor of LPG (in hybrids too) which is more economical, and does not emit much pollution. probably make all hybrids pluggable, and allow vehicles(new) to be offered in non-hybrid version only alongside an hybrid version of the same model (with a publicly-funded incentive that reduce to price/cost gap between the two versions to a maximum that reduces over the next, say 11 years or so), in the first few years allow exception for some models. but tax these with a luxury tax (so that it makes sense to look for a model where you consider an hybrid).
